### PR TITLE
animated card stack animation added

### LINKED ISSUE #102: Animated card stacks
### Feature request
I would like to add animated cards stacks.

### Is your feature request related to a problem? Please describe. (optional)
A clear and concise description of what the problem is.

### Describe how this feature will be useful to the users/ contributors-
A clear and concise description of how this feature will be useful to our users/ contributors.

#### Are you contributing under any open-source program ?

LGM-SOC
<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
- It is adviced to take up/ work on only 2 issues at the same time.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.
animated-card-stacks

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
Added the animated card stack animation

## Related issue?
Fixes #102 

## How was it tested?
Remove this line and tell us how the code was tested.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

-   [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
-   [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/Mirage-UI/pulls) for the same update/change?
-   [x] Have you made corresponding changes to the documentation?
-   [x] Have you tested the code before submission?
-   [x] Did you lint and format your code before making the Pull Request?
-   [x] Have you made the GIF for the component you added?

GIF
animated

### CODE REVIEW

File: HTML+CSS+JS/Cards/AnimatedCardStacksAnimation/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Animated Card stack
+
```
Comment (Pranav016) [inline HTML+CSS+JS/Cards/AnimatedCardStacksAnimation/README.md:2]: You need to add a demo GIF or video for your component, take reference from this-
https://github.com/ALPHAVIO/Mirage-UI/tree/master/React/Navigation-Bar/TopDownAnimation-ResponsiveNavbar

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Also update the index with the new component you are adding - https://github.com/ALPHAVIO/Mirage-UI/blob/master/HTML%2BCSS%2BJS/README.md

**Comment by Khushi-agarwal** [comment]
@Pranav016 done the changes.Kindly review them.

**Review by theBeginner86 — APPROVED** [review]
Everything looks good now. 👍

**Comment by Pranav016** [comment]
@all-contributors please add @Khushi-agarwal for code

**Comment by Pranav016** [comment]
@all-contributors please add @Khushi-agarwal for doc

**Pull request merged**